src/debug.ts

```diff
@@ -0,0 +1,52 @@
+// The MIT License (MIT)
```

Comment:
I wonder if this should go into src/common since most of the stuff under src/ will go into v1 in the next major change.